sourcemash/api/items.py

```diff
@@ -10,6 +10,8 @@
 
 class getVote(fields.Raw):
     def output(self, key, item):
+        if not current_user.is_authenticated():
+            return 0      
```

Comment:
Sounds good.

So when a un-logged-in user clicks the saved button, we'll prompt them to sign up, and same for voting?